duckduckgo/Android
# Swap bookmarks and refresh menu buttons for discoverability

Asana Issue URL: https://app.asana.com/0/361428290920652/536546061737886

## Description
Swap bookmarks and refresh menu buttons for discoverability

## Steps to Test this PR:
1. Open the app
1. Ensure the Bookmarks and Refresh buttons are now swapped
